manifests/release/demo-content.yaml.in

```diff
@@ -13,6 +13,7 @@ spec:
     resources:
       requests:
         memory: 2G
+    devices: {}
```

Comment:
@fabiand I think that you needed that is also because of the wrong kubevirt version. Can you revert?